pandas/tools/util.py

```diff
@@ -61,6 +61,13 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : str, numpy.dtype, or Python dtype, default None
```

Comment:
> Also, what do you mean by "typical" usage?

I think Jeff means a "typical" way users would want to use this, not how it is typically done elsewhere in the codebase (for users, this is a new feature)